src/swarm-tui/Swarm/TUI/Model/Name.hs

```diff
@@ -110,4 +112,6 @@ data Name
     REPLViewport
   | -- | A clickable button in a modal dialog.
     Button Button
+  | -- | A custom widget name.
```

Comment:
Might be nice to elaborate a bit here to explain what this is for, especially since we will not be using this directly.

Reply:
I hope this is satisfactory.